zhaquirks/owon/Owon_PC321_Z.py

```diff
@@ -0,0 +1,99 @@
+"""QUIRK FOR OWON PC321 Z"""
+
+import logging
+
+from zigpy.profiles import zha
+from zigpy.quirks import CustomCluster, CustomDevice
+import zigpy.types as t
+from zigpy.zcl.clusters.general import Basic, Identify
+from zigpy.zcl.clusters.smartenergy import Metering
+
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class Owon_PC321_Z_Simple_Metering(CustomCluster, Metering):
+    """Owon PC321 CustomCluster"""
+
+    cluster_id = 0x0702
+    ep_attribute: str = "owon_smartenergy_metering"
```

Comment:
Hmm, so this will make the existing `Metering` sensors not match.
Looking at the [HA PR](https://github.com/home-assistant/core/pull/100735), `current_summ_delivered` and `instantaneous_demand` would be duplicated.

Since the quirk only adds new attributes (and doesn't change or remove any existing attributes), we should be able to keep the `ep_attribute` of `Metering` (and change the matching in HA for the new sensors to `CLUSTER_HANDLER_SMARTENERGY_METERING`, but also include the manufacturer/model of the device).

Reply:
The reason why a used this new ep_attribute was to create in zha/sensors new sensors especially for this Owon component.
I didn't face any problem.
Let me make some test, if I could maintain the Metering ep_attribute with new sensors and check if it's still working well

Reply:
I made changes for ep_attribute to smartenergy_metering.
Made changes in zha/sensor.py too, all is working well (it is the next PR to have all sensors from this device available)